xbmc/windowing/osx/WinEventsOSXImpl.mm

```diff
@@ -389,6 +315,41 @@ - (XBMC_Event)keyPressEvent:(NSEvent*)nsEvent
   return newEvent;
 }
 
+- (void)SendXBMCMouseEvent:(NSEvent*)
+                   nsEvent:(XBMC_Event&)xbmcEvent
+                          :(uint8_t)mouseEventType
+                          :(std::optional<uint8_t>)buttonCode
```

Comment:
- in 99% cases every parameter should have label, it makes the method call readable
- label should be on the left of colon, param name on the right

something like
```objc
- (void)SendXBMCMouseEvent:(NSEvent*)nsEvent
                 xbmcEvent:(XBMC_Event&)xbmcEvent
            mouseEventType:(uint8_t)mouseEventType
                buttonCode:(std::optional<uint8_t>)buttonCode
```

and the call will look like:
```objc
[self SendXBMCMouseEvent:nsEvent
               xbmcEvent:newEvent
          mouseEventType:XBMC_MOUSEBUTTONDOWN
              buttonCode:XBMC_BUTTON_MIDDLE];
```